tests/integration/docusaurus/tutorials/getting-started/getting_started.py

```diff
@@ -39,23 +39,43 @@
             - data_asset_name
           pattern: (.*)
 """
+
+# Note : this override is for internal GE purposes, and is intended to helps us better understand how the
+# Getting Started Guide is being used. It can be ignored by users.
```

Comment:
This is the message that is being used throughout this script... please have a look and let me know if any of it can be improved.